lib/iris/analysis/maths.py

```diff
@@ -45,19 +45,50 @@
 _output_dtype_cache = {}
 
 
-def _output_dtype(op, left_dtype, right_dtype=None, in_place=False):
-    operand_dtypes = [left_dtype, right_dtype] if right_dtype is not None \
-                      else [left_dtype]
+def _output_dtype(op, first_dtype, second_dtype=None, in_place=False):
```

Comment:
While writing the docstring I realised 'left' and 'right' didn't really make sense for the names of these arguments.